Makefile

```diff
@@ -15,11 +15,18 @@
 # limitations under the License.
 #
 
+export GO111MODULE=on
+
 ### license-check:    Check apisix-dashboard source codes for Apache License
 .PHONY: license-check
 license-check:
 ifeq ("$(wildcard .actions/openwhisk-utilities/scancode/scanCode.py)", "")
 	git clone https://github.com/apache/openwhisk-utilities.git .actions/openwhisk-utilities
 	cp .actions/ASF* .actions/openwhisk-utilities/scancode/
 endif
-	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
\ No newline at end of file
+	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
+
+
+.PHONY: api-test
```

Comment:
please add a comment like this line: https://github.com/apache/apisix-dashboard/blob/90c11e50385b3b9dbde49c0b1c6e0f31c00995e0/Makefile#L20

then we can get the command list: `make help`